src/read_agent_logging.in

```diff
@@ -128,7 +129,7 @@ URL="${1:-http://localhost:24231/metrics}"
 exec curl --silent "${URL}" \
   | grep -E '^stackdriver_' \
   | sed -E \
-    -e "s/^stackdriver_([a-zA-Z0-9_]*)(\{(.*code=\"([0-9]*)\")?.*\})? ([0-9]*)/PUTVAL ${COLLECTD_HOSTNAME}\/agent-\4\/derive-\1 N:\5/" \
+    -e "s/^stackdriver_([a-zA-Z0-9_]*)(\{(.*code=\"([0-9]*)\")?.*\})? ([0-9]*).*/PUTVAL ${COLLECTD_HOSTNAME}\/agent-\4\/derive-\1 N:\5/" \
```

Comment:
By fractional part, do you mean it'll have a decimal? If so, can you use `(\..*)?` ?

Reply:
Done, although a relaxed parser like in my original change has better future proofing.